### ISSUE TITLE
IR Gun Control Settings not working on Wii

### ISSUE THREAD

**Issue by CrispyAngels** [emukidid/pcsxgc#45]
Settings for gun mode in control settings not allowing customization of ir functionality. Games do not detect wiimote ir on screen. Suggest modification of settings for this mode.

**Comment by emukidid** [comment]
Yup, gun support is non-existent right now, I need to add it back in.

**Comment by CrispyAngels** [comment]
Well there still is "Gun" as a control type in the latest build. Not really functional though. Suggest changing mapping options for these controls. Might of just been an oversight.

**Comment by emukidid** [comment]
The latest build adds back gun support across controller types but still needs some further settings around it all. Happy to have some feedback if you get some time to try it.

**Comment by emukidid** [comment]
Fixed in the next release.
